client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -104,8 +104,15 @@ export class CheckoutThankYouHeader extends PureComponent {
 				<>
 					<div>
 						{ _n(
-							"We got it from here! We'll send an email when your domain is ready to use.",
-							"We got it from here! We'll send an email when your domains are ready to use.",
+							'We got it from here. Your domain is being transferred with no downtime.',
+							'We got it from here! Your domain is being transferred with no downtime.',
```

Comment:
Should this be "We've got it from here"?
```suggestion
							'We've got it from here! Your domain is being transferred with no downtime.',
```

Reply:
Just pushed a commit to address this. Will plan on deploying afterwards.